.github/workflows/build-image.yml

```diff
@@ -0,0 +1,32 @@
+# Triggered by Pull Request or Manually (from GitHub UI) events
+
+name: Publish image to Registry
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Push to Registry
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out code
+      uses: actions/checkout@v2.3.1
+
+    - name: Prepare tag
+      id: prepare
+      run: |
+        commit_id=$(git rev-parse --short HEAD)
+        branch_name=${{ github.head_ref }}
+        tag="${branch_name}-${commit_id}"
+        echo "::set-output name=tag::$tag"
+
+    - name: Push Atlas-osb to Registry
+      uses: docker/build-push-action@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+        repository: ${{ secrets.DOCKER_REPO }}
```

Comment:
do you think it's worth adding the default `.actrc` file with correct values for local repo to the project? Or maybe the template file (so the engineers could make their own `.actrc`)